<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>FormOrganigrama</class>
  <widget class="QWidget" name="FormOrganigrama">
    <property name="windowTitle">
      <string>Formulario Organigrama</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
            <widget class="QLabel" name="titulo_label">
              <property name="text">
                <string>TÃ­tulo:</string>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QLineEdit" name="titulo_lineEdit"/>
          </item>
        </layout>
      </item>
      <item>
        <widget class="QPushButton" name="enviar_button">
          <property name="text">
            <string>Enviar</string>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>
